### PR TITLE
FTB Quests

### PR DESCRIPTION
**Pull request by Gaz492**

### CODE REVIEW

File: docs/mods/suite/Quests/Developer/Integrations/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+FTB Quests has a number of integrations with other mods. These integrations allow you to use the features of other mods in your quests.
+
+:::caution
+WIP
+:::
+
+FTB Filter system but that’s a mod that needs it’s own docs anyway. Basically, complex filters that allows authors to configure item checks via a GUI
+
+JEI, REI: Works with lib as well and shows a dedicated page on the item selection for JEI items, it also adds quest items as a reward section in the uses / recipe tabs, it might do more, not sure.
```
Comment (desht) [inline docs/mods/suite/Quests/Developer/Integrations/index.md:14]: Also loot crates - if a reward table has a loot crate, then JEI/REI will show what items drop from that loot crate.

File: docs/mods/suite/Quests/Developer/Integrations/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+KubeJS: Events and custom quest types as well as custom filters
+
+Not sure if there is anything else par from those
```
Comment (desht) [inline docs/mods/suite/Quests/Developer/Integrations/index.md:18]: Nope, that's about it. Filtering, KubeJS, and JEI/REI.

File: docs/mods/suite/Quests/Developer/Quests/Rewards.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+title: Quest Rewards
+sidebar_position: 3
+---
+
+| Type          | Description                                             |
+|---------------|---------------------------------------------------------|
+| Item          | Give an item as a reward                                |
+| Choice Reward | Allow the player to choose from a selection of rewards  |
+| Random Reward | The player gets a random reward from a set reward table |
+| Loot Reward   | Reward a player with loot from a reward table           |
```
Comment (desht) [inline docs/mods/suite/Quests/Developer/Quests/Rewards.md:11]: Might be worth documenting the distinction between "random" and "loot" reward. They both reward a random item from a reward table - the difference is that if the reward table has a chance to drop nothing, then "loot" rewards can reward nothing, while "random" rewards will always reward an item.

File: docs/mods/suite/Quests/Developer/Quests/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+:::info
+To create a quest you will first need to create a chapter, a quest is always created inside a chapter and a chapter can
+be put inside a chapter group.
+
+For details on creating a chapter, see the [Chapters](../Chapters/index.md) page.
+:::
+
+To create a quest, right anywhere in the blank space of the quest book and select the type of quest you want to create.
```
Comment (desht) [inline docs/mods/suite/Quests/Developer/Quests/index.md:13]: Strictly, the type of _task_ you want to create... it creates a default quest with one task added to it.

File: docs/mods/suite/Quests/Developer/Styling/index.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+Firstly: Themes are just a `.txt` file that tells quests how to look. Most components will use this for their styling. If they don't. Let us know! Not many people use this system
+
+Secondly: Themes are handled by placing a `ftbquests/ftb_quests_theme.txt` file in your resource pack (kubejs works here)
+
+## The fun stuff
+
+### Properties
+
+#### Color
+- `color`: A colo(u)r property is a RGBA colour code but it supports a bunch of things so here goes
```
Comment (desht) [inline docs/mods/suite/Quests/Developer/Styling/index.md:15]: "colo(u)r" isn't really necessary here.  Just pick US or GB english and be consistent :stuck_out_tongue:

File: docs/mods/suite/Quests/Player/Questbook/Opening.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+The quest book is a simple and easy-to-use way to track your progress in the game. It is divided into chapters, each of which contains a number of quests. Each quest has a title, a description, and a set of objectives that you need to complete in order to finish it. When you complete a quest, you will be rewarded with items, experience, or other rewards.
+
+## How to open the quest book
+
+You can open the quest book by opening your inventory and clicking on the book icon in the top left of the screen. This will open the quest book interface, where you can see all of the quests and chapters that are available.
+
+![Opening the quest book](../../../../_assets/images/quests/opening-questbook-inv.webp)
+
+There is also a quest book item that you can craft and use to open the quest book interface.
```
Comment (desht) [inline docs/mods/suite/Quests/Player/Questbook/Opening.md:14]: Should also mention that there's a keybind for opening the quest book.

**Comment by Jake-E** [comment]
@Gaz492 How're we looking with this?

**Review by Jake-E — APPROVED** [review]

**Pull request merged**